source/postprocess/geoid.cc

```diff
@@ -989,27 +1080,17 @@ namespace aspect
       {
         prm.enter_subsection("Geoid");
         {
-          include_dynamic_topo_contribution = prm.get_bool ("Include the contributon from dynamic topography");
+          include_topo_contribution = prm.get_bool ("Include the contributon from dynamic topography"); // This parameter doesn't exist anymore (incl. spelling error). TODO: throw error message when used.
```

Comment:
If you remove the declaration of this parameter from the header file you have to declare it here: `const bool include_topo_contribution = ...`.

Add the following line to address the comment (you may need to wrap that over several lines):
```
AssertThrow (include_topo_contribution == true, 
                      ExcMessage("The parameter 'Include the contributon from dynamic topography' has been replaced by the two parameters 'Include the contribution from surface topography' and 'Include the contribution from CMB topography', please use them instead.);
```